src/main/java/paymelah/logic/commands/ListDebtorsCommand.java

```diff
@@ -26,6 +27,7 @@ public class ListDebtorsCommand extends Command {
     public static final String MESSAGE_SUCCESS_SPECIFIC = "Listed all persons with debts amounting to more than $%1$s.";
     public static final String MESSAGE_SUCCESS_GENERIC = "Listed all persons with debts.";
     public static final String MESSAGE_NO_DEBTORS = "There are no persons with debts";
+    public static final Money ZERO_MONEY = new Money("0");
```

Comment:
Should this be a constant of Money instead of ListDebtorsCommand?

Reply:
ok will change